src/components/ResourceTypeSelector/ResourceTypeSelector.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from 'react';
+import {
+  StyleSheet, Text, ScrollView,
+} from 'react-native';
+import { Button } from 'native-base';
+import {
+  bool, func, shape, string,
+} from 'prop-types';
+import { connect } from 'react-redux';
+
+import { supportedResourceTypeFiltersSelector } from '../../redux/selectors';
+import Colors from '../../constants/Colors';
+import RESOURCE_TYPES from '../../resources/resourceTypes';
+import { selectResourceType } from '../../redux/epics';
+
+const ResourceTypeSelector = ({ resourceTypeFilters, selectResourceTypeAction }) => {
+  useEffect(() => {
```

Comment:
I don't think it's necessary to use `useEffect` for this feature...  the default date is implicit and derivable, e.g.: as a default state in a selector.

Reply:
there are several reasons, e.g.: we need worry less about whether `ResourceTypeSelector` stays mounted.

Reply:
This was created because I wanted at least one `resourceType` selected on the `CatalogScreen`. Additionally this should be a `resourceType` that was not filtered by the `supportedResourceTypeFiltersSelector`. So I built the `useEffect` downstream.

If not here, where should the selection be made?